db/migrate/20180329164621_change_all_enum_to_integer.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeAllEnumToInteger < ActiveRecord::Migration[5.1]
+  def change
+    remove_column :timeslots, :day
```

Comment:
Next time you can just use `change_column`

Reply:
I did but there was some error with casting :/